src/recipe.jl

```diff
@@ -44,3 +44,84 @@ end
 
     xs, ys
 end
+
+@recipe function g(sol::TMSol; vars=(0,1), timediv=1)
+    # Some initial checks
+    @assert timediv > 0 "`timediv must be 1 or larger"
+    @assert length(vars) == length(unique(vars)) == 2 "`vars` must have 2 distinct integer indices"
+    @assert 0 ≤ minimum(vars) && maximum(vars) ≤ get_numvars()
+
+    return _plot_intvbox(sol, vars=vars, timediv=timediv)
+end
+
+function _plot_intvbox(sol::TMSol; vars=(0,1), timediv=1)
+    domT = mince_in_time(sol, var=0, timediv=timediv)
+
+    if 0 ∈ vars
+        tup0 = findfirst(0 ∈ vars)
+        var1 = vars[3-tup0]
+        v1 = _mince_in_time(sol, domT, var1, timediv)
+        if tup0 == 1
+            return @. IntervalBox(domT, v1)
+        else
+            return @. IntervalBox(v1, domT)
+        end
+    end
+
+    var1, var2 = vars
+    v1 = _mince_in_time(sol, domT, var1, timediv)
+    v2 = _mince_in_time(sol, domT, var2, timediv)
+    return @. IntervalBox(v1, v2)
+end
+
+function mince_in_time(sol::TMSol; var::Int=0, timediv::Int=0)
+    domT = _mince_in_time(sol, Val(true), timediv)
+    var == 0 && return domT
+    return _mince_in_time(sol, domT, var, timediv)
+end
+
+# Mince in time var (var == 0)
+function _mince_in_time(sol::TMSol, ::Val{true}, timediv::Int=1)
+    # Case timediv == 1
+    if timediv == 1
+        return @. sol.time + domain(sol.xTM[1,:])
```

Comment:
Thanks a lot @mforets for the suggestions and the review. 

In the new commit I've tried to be more consistent... sorry for my consistent inconsistency. 😄 

Regarding the suggestion, you are right, I did chang the "convention" in the order of indices for `sol[indTime,indSolution]`, which corresponds to `sol.xTM[indSolution,indTime]`. Sorry for the mess that this may cause. The idea to do that was that `sol[indTime]` returns the whole vector of solutions by fixing the time index, which corresponds to `sol.time[indTime]`. I hope this makes things clearer now.

Regarding this: should the returned object be a `view`, or I leave it as it is now?

Reply:
Perfect :+1: I think it's good as it is now.